<?xml version="1.0"?>
<launch>
    <include file="$(find srs_assisted_arm_navigation)/launch/cob_planning_environment.xml" />
    <node pkg="ompl_ros_interface" type="ompl_ros" name="ompl_planning">
        <param name="default_planner_config" type="string" value="SBLkConfig1" />
        <rosparam command="load" file="$(find srs_assisted_arm_navigation)/config/ompl_planning.yaml" />
    </node>
</launch>
